pages/common/[.md

```diff
@@ -12,13 +12,9 @@
 
 `[ "${{variable}}" -{{eq|ne|gt|lt|ge|le}} {{integer}} ]`
 
-- Test if the specified variable has a [n]on-empty value:
+- Test if the specified variable has an empty[z]/[n]on-empty value:
```

Comment:
```suggestion
- Test if the specified variable has an empty or [n]on-empty value:
```
Mandatory fix: broken mnemonic must be removed, no word found contain `z` letter.
Optional fix: `or` may sound better.